pages/case-study-1.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <div>
+    <div class="feature__hero position-relative py-5">
+      <div class="container">
+        <div
+          class="d-flex flex-column justify-content-center align-items-center text-center mt-5"
+        >
+          <h1 class="hero__heading pt-3">
+            Uncovering Formester's Potential for
+            <span class="hglt">Lead Generation</span> Success
+          </h1>
+          <p class="hero__subheading mt-3">
+            In this case study, we'll explore how your B2B company can leverage
+            Formester to create and optimize lead generation forms, leading to
+            significant improvements in conversions and sales. This case study
+            is designed to provide insights into the strategies and tactics that
+            your organization can employ to achieve enhanced lead generation
+            results.
+          </p>
+          <a
+            href="https://app.formester.com/users/sign_up"
+            class="btn button hero__button mt-4 mb-4 mb-lg-0"
+            >Create Form for Free</a
+          >
+        </div>
+      </div>
+      <img
+        :src="require(`@/assets/images/case-study-1/case-study-hero-bg.png`)"
+        alt="background showing few of the ui elements of formester"
+        class="hero__background position-absolute d-none d-lg-block"
+      />
+    </div>
+
+    <CaseStudyOneBackground />
+    <SmallIconsFeatureSection :featureData="challenges" />
+    <SmallIconsFeatureSection :featureData="solutions" />
```

Comment:
Props can be named better. It can simple be `features`

Reply:
@piyushsinghania I believe featureData make sense here because there are many things that are present in the object. And I don't feel separating it out and passing too many props would I good idea here. 
![image](https://github.com/formester/formester-site/assets/59838143/4ca9643b-2e84-46e4-b9d1-421de44707f3)